changelog.json

```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "1.4.0-beta1": [
+      "[New] When an update is available for GitHub Desktop, the release notes can be viewed in Desktop - #2774",
+      "[New] Detect merge conflicts when comparing branches - #4588",
+      "[Fixed] Avoid double checkout warning when opening a pull request in Desktop - #5375",
+      "[Fixed] Clear error in Publish repository dialog on change of tab - #5422. Thanks @Daniel-McCarthy!",
```

Comment:
It may be worth noting this text is accurate to the original implementation. However, the functionality that was merged in allows each of the tabs to have their own errors and display the correct one for the appropriate tab.

This difference may not be too significant, however I thought it may be worth mentioning for accuracy sake.

Reply:
@Daniel-McCarthy I gave it another pass, let me know if that feels better

Reply:
@shiftkey Looks great! Thank you for updating that.